authentication/account/worker/user_deactivation_worker.go

```diff
@@ -37,10 +30,10 @@ func NewUserDeactivationWorker(ctx context.Context, app application.Application)
 }
 
 type userDeactivationWorker struct {
-	worker.Worker
+	worker.BaseWorker
 }
 
-func (w *userDeactivationWorker) deactivateUsers() {
+func (w userDeactivationWorker) deactivateUsers() {
```

Comment:
Any particular reason not to use a pointer?

Reply:
it's not needed